var/spack/repos/builtin/packages/bmi/package.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+from subprocess import call
+
+
+class Bmi(AutotoolsPackage):
+    """a communications framework and network abstraction layer"""
+
+    homepage = 'http://git.mcs.anl.gov/bmi.git/'
+    git = 'git://git.mcs.anl.gov/bmi'
+
+    version('develop', branch='master')
+
+    depends_on('autoconf', type='build')
+
+    # need to override 'autoreconf' so we can run BMI's 'prepare' script
+    def autoreconf(self, spec, prefix):
+        with working_dir(self.configure_directory):
+            call("./prepare")
```

Comment:
You can probably just use `Executable(./prepare)` from spack's imports here :)

Reply:
Thanks for the tip @ax3l .  Just pushed an update to the PR to do this.

Reply:
Whoops my apologies!  I guess I did not push that change when I said I did.  It's in place now, though.